src/snowcli/utils.py

```diff
@@ -505,6 +506,29 @@ def path_resolver(path_to_file: str):
     return path_to_file
 
 
+def check_if_package_is_avaiable_in_conda(package: Requirement, packages: dict) -> bool:
+    if package.name.lower() not in packages:
+        return False
+
+    if package.specs:
+        latest_ver = packages[package.name.lower()]["version"]
+        return _compare_specs(package.specs, latest_ver)
+
+    return True
+
+
+def _compare_specs(specs: List, latest: str):
```

Comment:
Maybe there's something in pip / build / setup package that can be used to do the comparison?

Reply:
Done